### PR TITLE
Updated design thinking + activity diagram

### PR DESCRIPTION
**Pull request by Aiswarya668**
#17 

Updated the design thinking approach in light of recommended changes and added the edited activity diagram as discussed incall

### CODE REVIEW

**Comment by Aiswarya668** [comment]
#14 
Updated the assumptions part as well to be more specific

**Comment by Aiswarya668** [comment]
There was a typo I just fixed as well

File: Report.md [hunk 1]
```diff
@@ -27,14 +27,15 @@ Problem Definition
...
-* Assume a supporting tool to create customer profiles exists, Profiler Tool.
-* Customer database already exists with information such as postcode, surnames
-* Employees have the appropriate product knowledge and understand the operation of the call management centre
-* Guidelines and a script exist for RM with outbound calls to assist them
+In the duration of this project the following assumption's were considered in all system analysis and feature recommendations. 
+
+* The CMC has enough processing power to handle complex computations and algorithms.
+* The CMC is already equipped with all data with information such as a customer database with information such as postcodes and surnames.
+* Employee’s within Major Travel Company have appropriate product knowledge and understanding of the operations within the CMC.
+* The functionality of the inbuilt profiler tool includes customer profile generation and allows the CMC to auto-generate a generic list of customers for specified Relationship Manager’s (RM’s).
```
Comment (zachartmann) [inline Report.md]:
```suggestion
* The functionality of the inbuilt profiler tool includes customer profile generation and allows the CMC to auto-generate a generic list of customers for specified Relationship Managers (RMs).
```

File: Report.md [hunk 2]
```diff
@@ -72,7 +73,28 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+
+### Empathise
+Phase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives, empathy maps were utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution.
```
Comment (zachartmann) [inline Report.md]: Sounds great, just one or two things to make it easier to read
```suggestion
Phase one consisted of empathising with stakeholders. This represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step in achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who will be affected. To document these perspectives, empathy maps were utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
```

File: Report.md [hunk 3]
```diff
@@ -72,7 +73,28 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+
+### Empathise
+Phase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives, empathy maps were utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
+
+### Define
+Phase two of defining allowed us to synthesise the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given.
```
Comment (zachartmann) [inline Report.md]: "Phase two of defining"
Phase two of design thinking _is_ defining?
Also "This stage" not staged

File: Report.md [hunk 4]
```diff
@@ -72,7 +73,28 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+
+### Define
+Phase two of defining allowed us to synthesise the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 
+
+### Ideate
+Phase three consisted of utilising the user research conducted to ideate solutions to the problem statements we created in the prior stage. The brainstorming and worst possible idea techniques were utilised to assist within our ideation stage. These techniques were chosen to initiate freeform group discussions and simulated creative solutions in light of the problems identified. These sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. The conclusions of the brainstormed features were documented in into issues with git. At the conclusion of this stage, the agreed features were converted to issues to be developed for the prototype and testing stage.
```
Comment (zachartmann) [inline Report.md]:
```suggestion
Phase three consisted of utilising the user research conducted to ideate solutions to the problem statements we created in the prior stage. The brainstorming and 'worst possible idea' techniques were utilised to assist within our ideation stage. These techniques were chosen to initiate freeform group discussions and simulated creative solutions in light of the problems identified. These sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. The conclusions of the brainstormed features were documented into git via issues. At the conclusion of this stage, the agreed-upon features were converted to issues to be developed for the prototype and testing stage. 
```

File: Report.md [hunk 5]
```diff
@@ -72,7 +73,28 @@ Stakeholders refer to individuals/entities that have an interest in the developm
...
+
+### Prototype/Test
+The prototype and subsequent testing phase is representative of the final stages within the design thinking approach. This consists of producing a small-scaled, mock-up of the product or specific features found within the product so that tangible investigations can occur on the solutions generated in the previous stage. To assist in the prototype stage the team designed artefacts representative of the design of the system. The user flows are demonstrated within the use case diagram, activity diagrams, class diagram and collaborative diagrams provided. Due to the nature of this project, the prototype and respective testing are not within scope. 
+
+### Artefacts
+Artefact | Description
+---|---
+Empathy Maps | Is a tool used to visualise user attitudes and behaviours. This map is split into 6 quadrants (Says, Thinks, Does, Feels, Pains and Gains).
```
Comment (zachartmann) [inline Report.md]: You say "Is a tool" here but all other artefacts start with the name of the artefact in their description. Just for consistency

**Review by zachartmann — CHANGES_REQUESTED** [review]
Very slight changes.
Awesome work as always

File: Report.md (hunk 1 above)
Comment (Aiswarya668) [reply Report.md]: Yep will change that

File: Report.md (hunk 5 above)
Comment (Aiswarya668) [reply Report.md]: Yep I'll change that

File: Report.md (hunk 3 above)
Comment (Aiswarya668) [reply Report.md]: Also "This stage" not staged - totally didn't catch that  will change 
Phase two of design thinking is defining? Yea it is the wording if off will change it

**Comment by Aiswarya668** [comment]
Cool all adopted changes are in place - there shouldn't be anything left to change hopefully

**Review by zachartmann — APPROVED** [review]
Changes were good. No issues, well done

**Pull request merged**